procedures/foreman/release.md.erb

```diff
@@ -128,6 +128,9 @@ Note: If for some reason there was an issue with the tarballs that required uplo
 <% end -%>
 <% if full_version.end_with?('.0') %>
 - [ ] Update the `foreman_version` to <%= short_version %> in website's [`_config.yml`](https://github.com/theforeman/theforeman.org/blob/gh-pages/_config.yml) file
+- [ ] Update the [docs.theforeman.org](https://github.com/theforeman/foreman-documentation) [index page](https://github.com/theforeman/foreman-documentation/blob/master/web/content/index.adoc) and [versions](https://github.com/theforeman/foreman-documentation/blob/master/web/content/js/versions.js),
+  - [ ] Update <%= short_version %> as `supported`
+  - [ ] Update <%= (short_version.to_f - 0.2).round(2) %> as `unsupoorted`
```

Comment:
I'm not sure I like doing math this way in templates. Perhaps it should be created as an additional context here:
https://github.com/theforeman/tool_belt/blob/f0f9f35b3a8bab5c0ca34915ab07cf7be478628d/lib/tool_belt/commands/procedure.rb#L51-L73=

We already have major and minor as proper variables there. For example, your version will break when we get to Foreman 3.10: it'll show `2.9` rather than `3.8`

Also, there's a typo:
```suggestion
  - [ ] Update <%= (short_version.to_f - 0.2).round(2) %> as `unsupported`
```